h2o-bindings/bin/custom/python/gen_infogram.py

```diff
@@ -0,0 +1,323 @@
+            mask_valid = admissible_valid > 0       
+        
+        if xval:
+            X_xval = np.array(rel_cmi_frame_xval[ig_x_column].as_data_frame(header=False, use_pandas=False)).astype(float).reshape((-1,))
+            Y_xval = np.array(rel_cmi_frame_xval[ig_y_column].as_data_frame(header=False, use_pandas=False)).astype(float).reshape((-1,))
+            features_xval = np.array(rel_cmi_frame_xval[features_column].as_data_frame(header=False, use_pandas=False)).reshape((-1,))
+            admissible_xval = np.array(rel_cmi_frame_xval[index_of_admissible].as_data_frame(header=False, use_pandas=False)).astype(float).reshape((-1,))
+            mask_xval = admissible_xval > 0
+
+        plt.figure(figsize=figsize)
+        plt.grid(True)
+        plt.scatter(X, Y, zorder=10, c=np.where(mask, "black", "gray"), label="training data")
+        if valid:
+            plt.scatter(X_valid, Y_valid, zorder=10, marker=",", c=np.where(mask_valid, "black", "gray"), label="validation data")
+        if xval:
+            plt.scatter(X_xval, Y_xval, zorder=10, marker="v", c=np.where(mask_xval, "black", "gray"), label="xval holdout data")
+        if legend_on:
+            plt.legend(loc=2, fancybox=True, framealpha=0.5)
+        plt.hlines(y_thresh, xmin=x_thresh, xmax=xmax, colors="red", linestyle="dashed")
+        plt.vlines(x_thresh, ymin=y_thresh, ymax=ymax, colors="red", linestyle="dashed")
+        plt.gca().add_collection(PolyCollection(verts=[[(0,0), (0, ymax), (x_thresh, ymax), (x_thresh, y_thresh), (xmax, y_thresh), (xmax, 0)]],
+                                                color="#CC663E", alpha=0.1, zorder=5))
+        
+        for i in mask.nonzero()[0]:
+            plt.annotate(features[i], (X[i], Y[i]), xytext=(0, -10), textcoords="offset points",
+                         horizontalalignment='center', verticalalignment='top', color="blue")
+        
+        if valid:
+            for i in mask_valid.nonzero()[0]:
+                plt.annotate(features_valid[i], (X_valid[i], Y_valid[i]), xytext=(0, -10), textcoords="offset points",
+                             horizontalalignment='center', verticalalignment='top', color="magenta")
+
+        if xval:
+            for i in mask_xval.nonzero()[0]:
+                plt.annotate(features_xval[i], (X_xval[i], Y_xval[i]), xytext=(0, -10), textcoords="offset points",
+                             horizontalalignment='center', verticalalignment='top', color="green")
+        
+        plt.xlim(0, 1.05)
+        plt.ylim(0, 1.05)
+        plt.xlabel(x_label)
+        plt.ylabel(y_label)
+        plt.title(title)
+        fig = plt.gcf()
+        if not server:
+            plt.show()
+        return fig
+        
+    def get_relevance_cmi_frame(self, valid=False, xval=False):
+        """
+        Retreive relevance, cmi information in an H2O frame for training dataset by default
```

Comment:
In the docstrings, I would be a little more verbose - if there is some abbreviation I think we could use the unabbreviated form at least on the first usage, e.g., instead of `cmi` I'd use `conditional mutual information (cmi)`. (Assuming it is the conditional mutual information). 

I'm not sure if `cmi` is well-known abbreviation but to me as a non-native English speaker this is unknown and I think this could save some time for the users.

Reply:
Yeah, the paper used to use "CMI",  but the terminology has changed since the early versions of the paper.  However, the backend Java code has not been updated and uses "CMI", so I am wondering if these client-side utility functions should be updated as well... Maybe something like `get_admissibile_information`?  or `get_admissible_score()` since the data frame object is called `admissible_score`?  @wendycwong

Reply:
@wendycwong and I agreed that I will come up with a mapping of old cmi function names to new names and we will update this.

Reply:
These have been updated.